.github/workflows/nightly-build.yaml

```diff
@@ -0,0 +1,36 @@
+name: Nightly
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Every day at midnight
+
+jobs:
+
+  build:
+    name: build
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
does it always check out the master branch?

Reply:
Whoops, i removed the branch specification when i removed the pull-request trigger i was using for testing.

Reply:
Yea, the schedule trigger works on the default/master branch.

Here is a discussion i found about this.
https://github.community/t/on-schedule-per-branch/17525

Reply:
actions/checkout will checkout whatever ref triggered the action (you can override it tho).
The on->schedule action currently only triggers against the default branch in the repo.